ce/ce/util/safeEval.ts

```diff
@@ -25,3 +25,116 @@ export function createSandbox(): <T>(code: string, context?: any) => T {
     return sandbox[response];
   };
 }
+
+export const safeEval = createSandbox();
+
+function isValue(cursor: Scanner) {
```

Comment:
where are these functions called? How do we know what's being passed to `safeEval`?

Reply:
`safeEval` evaluates `is` statements. `isValue` helps determine if these statements are valid expressions.